apis/nucleus/src/components/listbox/interactions/keyboard-navigation/keyboard-nav-rows.js

```diff
@@ -75,12 +75,7 @@ export default function getRowsKeyboardNavigation({
         }
 
         // Without shift key
-        if (inSelection) {
-          keyboard.focusSelection();
-        } else {
-          currentTarget.blur();
-          keyboard.blur();
-        }
+        keyboard.focusSelection();
```

Comment:
There is an issue  with this function `keyboard.focusSelection();` which doesn't return anything so we don't know if there is anything that get focused after this function is called. It would be good if this function does return the focused element. Is it ok to do that? There was a API spec for this function saying not return anything.

Reply:
I think that would be fine, if you really need it. How come you need to know which elements get focused?